### PR TITLE
remove human health category, update copy

### PR DESCRIPTION
**Pull request by zhangium**
- Removed Human Health category
- Updated copy to reflect this change

### CODE REVIEW

File: src/components/ProductBarInfoPage.js [hunk 1]
```diff
@@ -52,9 +52,9 @@ export const ProductBarInfoPage = () => {
   const classes = useStyles();
   const score = calculateScore(randomInfo);
   let colour;
-  if (score >= 10) {
+  if (score >= 8) {
```
Comment (wongcmichelle) [inline src/components/ProductBarInfoPage.js]:
```suggestion
  if (score >= 9) {
```

According to our team messenger chat: 0-4 bad, 5-8 ok, 9-12 good

File: src/components/ProductBarInfoPage.js [hunk 2]
```diff
@@ -52,9 +52,9 @@ export const ProductBarInfoPage = () => {
...
     colour = Colours.Success;
-  } else if (score >= 5) {
+  } else if (score >= 4) {
```
Comment (wongcmichelle) [inline src/components/ProductBarInfoPage.js]:
```suggestion
  } else if (score >= 5) {
```

File: src/utils/data.ts [hunk 3]
```diff
@@ -9,10 +9,6 @@ export const randomInfo: ProductInfo = {
     score: 2,
     details: [],
   },
-  humanHealth: {
-    score: 3,
-    details: [],
-  },
   supplyChainAndEnvironment: {
```
Comment (wongcmichelle) [inline src/utils/data.ts]:
```suggestion
  manufacturingEmissions: {
```

File: src/utils/data.ts [hunk 4]
```diff
@@ -37,13 +33,6 @@ export const squalaneCleanserInfo: ProductInfo = {
...
     ],
   },
-  humanHealth: {
-    score: 2,
-    details: [
-      "All Deciem formulations are made by in-house team of bio-chemists and material-chemists in their own labs",
-      "All Deciem products undergo thorough safety assessment before hitting the market. Each ingredient is reviewed against current scientific literature and strict global regulations",
-    ],
-  },
   supplyChainAndEnvironment: {
```
Comment (wongcmichelle) [inline src/utils/data.ts]:
```suggestion
  manufacturingEmissions: {
```

File: src/utils/data.ts [hunk 5]
```diff
@@ -71,13 +60,6 @@ export const soyMakeupRemovingFaceWashInfo: ProductInfo = {
...
     details: ["Fresh is against and does not use animal testing"],
   },
-  humanHealth: {
-    score: 2,
-    details: [
-      "This product contains Rosa Damascena Flower Oil which is expensive to extract and is required in large amounts, also contains fragrant components that might irritate sensitive skin",
-      "This product used Ascorbyl Palmitate (AP), which is a not so stable vitamin C derivative. AP can promote lipid peroxidation and cytotoxicity",
-    ],
-  },
   supplyChainAndEnvironment: {
```
Comment (wongcmichelle) [inline src/utils/data.ts]:
```suggestion
  manufacturingEmissions: {
```

File: src/utils/data.ts [hunk 6]
```diff
@@ -104,12 +86,6 @@ export const superfoodAntioxidantCleanserInfo: ProductInfo = {
     score: 3,
     details: ["Never, the company’s line is 100% vegan and cruelty free"],
   },
-  humanHealth: {
-    score: 2,
-    details: [
-      "All ingredients in this product are biodegradable and abide by EU’s list of 1300+ ingredients that are restricted for cosmetic use",
-    ],
-  },
   supplyChainAndEnvironment: {
```
Comment (wongcmichelle) [inline src/utils/data.ts]:
```suggestion
  manufacturingEmissions: {
```

Comment (wongcmichelle) [reply src/utils/data.ts]: Just to be consistent with cosmelo-ext-react

File: src/utils/helper.ts [hunk 7]
```diff
@@ -3,7 +3,6 @@ import { Colours } from "./Constants";
 export enum ScoreCategories {
   Packaging = "packaging",
   Disclosure = "disclosure",
-  HumanHealth = "humanHealth",
   SupplyChainAndEnvironment = "supplyChainAndEnvironment",
```
Comment (wongcmichelle) [inline src/utils/helper.ts]:
```suggestion
  ManufacturingEmissions = "manufacturingEmissions",
```

File: src/utils/helper.ts [hunk 8]
```diff
@@ -3,7 +3,6 @@ import { Colours } from "./Constants";
 export enum ScoreCategories {
   Packaging = "packaging",
   Disclosure = "disclosure",
```
Comment (wongcmichelle) [inline src/utils/helper.ts]:
```suggestion
  AnimalTesting = "animalTesting",
```

File: src/utils/helper.ts [hunk 9]
```diff
@@ -16,7 +15,6 @@ export interface CategoryInfo {
 export interface ProductInfo {
   packaging: CategoryInfo;
   disclosure: CategoryInfo;
```
Comment (wongcmichelle) [inline src/utils/helper.ts]:
```suggestion
  animalTesting: CategoryInfo;
```

File: src/utils/helper.ts [hunk 10]
```diff
@@ -16,7 +15,6 @@ export interface CategoryInfo {
...
-  humanHealth: CategoryInfo;
   supplyChainAndEnvironment: CategoryInfo;
```
Comment (wongcmichelle) [inline src/utils/helper.ts]:
```suggestion
  manufacturingEmissions: CategoryInfo;
```

File: src/utils/helper.ts [hunk 11]
```diff
@@ -27,10 +25,6 @@ export const categories = {
     colour: Colours.Skobeloff,
   },
   disclosure: { label: "Animal Testing", colour: Colours.SkyBlueCrayola },
```
Comment (wongcmichelle) [inline src/utils/helper.ts]:
```suggestion
  animalTesting: { label: "Animal Testing", colour: Colours.SkyBlueCrayola },
```

File: src/utils/helper.ts [hunk 12]
```diff
@@ -27,10 +25,6 @@ export const categories = {
...
-  humanHealth: {
-    label: "Human Health",
-    colour: Colours.NaplesYellow,
-  },
   supplyChainAndEnvironment: {
```
Comment (wongcmichelle) [inline src/utils/helper.ts]:
```suggestion
  manufacturingEmissions: {
```

File: src/utils/helper.ts [hunk 13]
```diff
@@ -44,7 +38,6 @@ export const categories = {
 export const orderedCategories: Array<keyof ProductInfo> = [
   ScoreCategories.Packaging,
   ScoreCategories.Disclosure,
```
Comment (wongcmichelle) [inline src/utils/helper.ts]:
```suggestion
  ScoreCategories.AnimalTesting,
```

File: src/utils/helper.ts [hunk 14]
```diff
@@ -44,7 +38,6 @@ export const categories = {
...
-  ScoreCategories.HumanHealth,
   ScoreCategories.SupplyChainAndEnvironment,
```
Comment (wongcmichelle) [inline src/utils/helper.ts]:
```suggestion
  ScoreCategories.ManufacturingEmissions,
```

File: src/utils/helper.ts [hunk 15]
```diff
@@ -62,12 +55,6 @@ export const scoringByCategory = {
     1: "The company participates in animal testing, but is actively looking for alternatives",
     0: "The company performs animal testing on ingredients/products to justify safety",
   },
-  [ScoreCategories.HumanHealth]: {
-    3: "The company participates in ongoing research to improve product safety in ingredients and risk assessment",
-    2: "The company discloses risk and safety information on our websites including full risk assessment of ingredients, risk assessment methodologies, and postmarket safety surveillance strategy/results",
-    1: "The company does not address product safety beyond legal and regulatory compliance",
-    0: "The product includes ingredients that are present in Cosmetic Ingredients Hotlist beyond a regulated amount",
-  },
   [ScoreCategories.SupplyChainAndEnvironment]: {
```
Comment (wongcmichelle) [inline src/utils/helper.ts]:
```suggestion
  [ScoreCategories.ManufacturingEmissions]: {
```

File: src/utils/helper.ts [hunk 16]
```diff
@@ -62,12 +55,6 @@ export const scoringByCategory = {
     1: "The company participates in animal testing, but is actively looking for alternatives",
```
Comment (wongcmichelle) [inline src/utils/helper.ts:55]: I can't comment my suggestion for line 52, but it should be changed to `[ScoreCategories.AnimalTesting]: {`

File: src/utils/data.ts [hunk 17]
```diff
@@ -71,13 +60,6 @@ export const soyMakeupRemovingFaceWashInfo: ProductInfo = {
     score: 3,
```
Comment (wongcmichelle) [inline src/utils/data.ts]: `disclosure` should be `animalTesting`

File: src/utils/data.ts [hunk 18]
```diff
@@ -37,13 +33,6 @@ export const squalaneCleanserInfo: ProductInfo = {
       " Deciem does not test on animals and only sells to customers in China through cross-border operations",
```
Comment (wongcmichelle) [inline src/utils/data.ts]: `disclosure` should be `animalTesting`

File: src/utils/data.ts [hunk 19]
```diff
@@ -9,10 +9,6 @@ export const randomInfo: ProductInfo = {
     score: 2,
```
Comment (wongcmichelle) [inline src/utils/data.ts:9]: `disclosure` should be `animalTesting`

**Review by wongcmichelle — COMMENTED** [review]
Looks good! Just need to update the scoring logic. Also if we want to be consistent with cosmelo-ext-react then we should make these changes (I tried to make suggestions for most of them but I might've missed some so you could do a find/replace):

- `supplyChainAndEnvironment` -> `manufacturingEmissions`
- `SupplyChainAndEnvironment` -> `ManufacturingEmissions`
- `disclosure` -> `animalTesting`
- `Disclosure` -> `AnimalTesting`

File: src/components/ProductBarInfoPage.js (hunk 1 above)
Comment (zhangium) [reply src/components/ProductBarInfoPage.js]: Right! I'll change that 💯

**Review by wongcmichelle — APPROVED** [review]
Just remembered the `data.ts` file isn't updated with the copy changes. Feel free to just delete that file since i don't think it's used here.

File: src/utils/helper.ts [hunk 20]
```diff
@@ -26,12 +24,8 @@ export const categories = {
...
   },
-  disclosure: { label: "Animal Testing", colour: Colours.SkyBlueCrayola },
-  humanHealth: {
-    label: "Human Health",
-    colour: Colours.NaplesYellow,
-  },
-  supplyChainAndEnvironment: {
+  animalTesting: { label: "Animal Testing", colour: Colours.SkyBlueCrayola },
+  manufacturingEmissions: {
     label: "Manufacturing greenhouse gas emissions",
```
Comment (wongcmichelle) [inline src/utils/helper.ts]: Sorry I missed this earlier 
```suggestion
    label: "Manufacturing Emissions",
```

**Pull request merged**